src/NHSD.BuyingCatalogue.Identity.Api/Views/Shared/_Layout.cshtml

```diff
@@ -1,4 +1,10 @@
-﻿<!DOCTYPE html>
+﻿@using Microsoft.Extensions.Configuration
+@inject IConfiguration config 
+@{
+    var urlPrefix = config.GetValue<string>("urlPrefix");
```

Comment:
The use of the configuration key `urlPrefix` is referenced in quite a few places. Would it be possible to abstract this configuration into an `ApplicationSettings` class or something? So that the configuration property only has to be changed in one place.